mobile/docker/Dockerfile

```diff
@@ -1,9 +1,7 @@
 # QT Installation Image --------------------------------------------------------
-FROM carlonluca/qt-dev:6.8.3
+FROM carlonluca/qt-dev:6.9.0
```

Comment:
I can help make a Dockerfile for this usecase, would prefer if we used one from our dockerhub

Reply:
in particular, rogue docker tags seems like an easy way to introduce a [supply chain attack](https://thehackernews.com/2025/06/malicious-pypi-npm-and-ruby-packages.html).

Reply:
Yes, removing untrustworthy sources of software is critical.